### DISCUSSION TITLE
images on documentation file not properly loaded

### DISCUSSION THREAD

**Discussion by adefaizal** [wavespectra/wavespectra#111] [Q&A] [answered]
Hi,

Firstly, I would like to thank you for this helpful package. It has been very useful to me. However, when I attempt to explore more features, it appears that the sample figure is not properly loaded in the documentation file.

![image](https://github.com/wavespectra/wavespectra/assets/138177594/23c16092-1e3a-4fed-a878-3dce57c66855)

and also in the web version

![image](https://github.com/wavespectra/wavespectra/assets/138177594/feb69d2b-1e14-4654-a521-873fc3441b38)

I am wondering if the developer could reload the documentation file properly.

Thank you.

Best regards,
Faizal

**Comment by rafa-guedes** [comment]
@adefaizal thank you for your feedback. The docs build is currently broken and there are some issues with the latest version available on readthedocs, including these missing images you have noticed. We are in the middle of some work on a new major release and the docs will be fixed with that, hopefully in the next few weeks. In the meanwhile, you could try and open one of the previous versions of the docs. At the bottom left of the docs page you you see a small `v:latest` link. If you click there you will see some options to view previous versions. `v3.15.1` for example should display the missing images. You can also access it from this link: https://wavespectra.readthedocs.io/en/v3.15.1/index.html

**Comment by rafa-guedes** [comment, marked as answer]
Docs are fixed now

https://wavespectra.readthedocs.io/en/latest/
